packages/maker/src/cli/main.ts

```diff
@@ -61,9 +70,21 @@ export default async function main(
       },
     })
     .parseAsync();
-
+  let apiNamespace = "";
+  if (commandLineOpts["api-namespace"].length !== 0) {
+    apiNamespace = (commandLineOpts["api-namespace"].slice(-1) !== ".")
+      ? commandLineOpts["api-namespace"] + "."
+      : commandLineOpts["api-namespace"];
+    invariant(
```

Comment:
can we also error out if the string is larger than some number, like 1024 characters? Otherwise there will be CVE filed about regexs DoS because thats life in 2024.

Reply:
Done